src/controllers/pageController.ts

```diff
@@ -158,11 +169,13 @@ async function myRequest(req: Request, res: Response) {
       );
       result.data.push({
         user_data: {
+          id: userFetchResult.data[0].id,
```

Comment:
nit: De-structure before used
```suggestion
          let {id, full_name, location, avatar} = userFetchResult.data;
          result.data.push({
          user_data: {
          id,
          full_name,
          location,
          avatar
          },
```